cmd/rpctest/rpctest/bench_ethgetblockbyhash.go

```diff
@@ -0,0 +1,115 @@
+package rpctest
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+// BenchEthGetBlockByHash compares response of Erigon with Geth
+// but also can be used for comparing RPCDaemon with Geth or infura
+// parameters:
+// needCompare - if false - doesn't call Erigon and doesn't compare responses
+//
+//			    false value - to generate vegeta files, it's faster but we can generate vegeta files for Geth and Erigon
+//	                 recordFile stores all eth_call returned with success
+//	                 errorFile stores information when erigon and geth doesn't return same data
+func BenchEthGetBlockByHash(erigonURL, gethURL string, needCompare, latest bool, blockFrom, blockTo uint64, recordFile string, errorFile string) {
+	setRoutes(erigonURL, gethURL)
+	var client = &http.Client{
+		Timeout: time.Second * 600,
+	}
+
+	var rec *bufio.Writer
+	var errs *bufio.Writer
+	var resultsCh chan CallResult = nil
+	var nBlocks = 0
+
+	if errorFile != "" {
+		f, err := os.Create(errorFile)
+		if err != nil {
+			fmt.Printf("Cannot create file %s for errorFile: %v\n", errorFile, err)
+			return
+		}
+		defer f.Close()
+		errs = bufio.NewWriter(f)
+		defer errs.Flush()
+	}
+
+	if recordFile != "" {
+		frec, errRec := os.Create(recordFile)
+		if errRec != nil {
+			fmt.Printf("Cannot create file %s for errorFile: %v\n", recordFile, errRec)
```

Comment:
Such error-handling looks "non-ideomatic in golang". 
Let's change signature of BenchEthGetBlockByHash func to return error. 
at higher level - `log.Error()` the error instead of printf. Use `fmt.Errof` to add info to error. 
configure logger - same as did in Erigon

`errRec` - usually errors just called `err`. 


`recordFile` - it's actually not a file - it's file name.

Reply:
OK
- I rename recordFile into recordFileName
- I rename errRec into err
- I rename errorFile into errorFileName 

I have seen that all the others bench* tests:
- doesn't have in the signature the return error
- they use printf instead of fmt.Errorf

Do you want modify this test or do You want modify all the tests ?

Reply:
"I have seen that all the others bench* tests" - i know. step-by-step will improve

Reply:
ok